cypress/locators/configurable-items/locators.js

```diff
@@ -1,3 +1,2 @@
 // component preview locators
-export const COMMON_INPUT_LABEL = 'label[data-element="label"]';
 export const ITEMS_WRAPPER = 'ol[data-element="configurable-items-wrapper"]';
```

Comment:
If we can remove 'ol' for ITEMS_WRAPPER, please remove it

```suggestion
export const ITEMS_WRAPPER = '[data-element="configurable-items-wrapper"]';
```

Reply:
updated